specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/examples/RedisEnterpriseSkuList.json

```diff
@@ -0,0 +1,608 @@
+                  "zoneMappings": [
+                    {
+                      "logicalZone": "1",
+                      "physicalZone": "uswest-AZ01"
+                    },
+                    {
+                      "logicalZone": "2",
+                      "physicalZone": "uswest-AZ02"
+                    }
+                  ],
+                  "locationCapabilities": [
+                    {
+                      "name": "ZoneAware",
+                      "value": true
+                    },
+                    {
+                      "name": "FDsAvailable",
+                      "value": true
+                    }
+                  ]
+                },
+                "skuDetails": {
+                  "SkuName": 9,
+                  "Name": "Enterprise_E1",
+                  "DefaultMaxMemory": 4294967296,
+                  "DefaultMaxFlash": 0,
+                  "PlanId": null,
+                  "InternalPlanId": null,
+                  "IsFlash": false,
+                  "IsLegacySku": false,
+                  "RequiresQuorumNode": true
+                }
+              },
+              {
+                "resourceType": "redisEnterprise",
+                "locationInfo": {
+                  "location": "West US",
+                  "zoneMappings": [
+                    {
+                      "logicalZone": "1",
+                      "physicalZone": "uswest-AZ01"
+                    },
+                    {
+                      "logicalZone": "2",
+                      "physicalZone": "uswest-AZ02"
+                    }
+                  ],
+                  "locationCapabilities": [
+                    {
+                      "name": "ZoneAware",
+                      "value": true
+                    },
+                    {
+                      "name": "FDsAvailable",
+                      "value": true
+                    }
+                  ]
+                },
+                "skuDetails": {
+                  "SkuName": 10,
+                  "Name": "Enterprise_E10",
+                  "DefaultMaxMemory": 11175000000,
+                  "DefaultMaxFlash": 0,
+                  "PlanId": "enterprise_10_000005",
+                  "InternalPlanId": "enterprise_10_000004",
+                  "IsFlash": false,
+                  "IsLegacySku": false,
+                  "RequiresQuorumNode": true
+                }
+              },
+              {
+                "resourceType": "redisEnterprise",
+                "locationInfo": {
+                  "location": "West US",
+                  "zoneMappings": [
+                    {
+                      "logicalZone": "1",
+                      "physicalZone": "uswest-AZ01"
+                    },
+                    {
+                      "logicalZone": "2",
+                      "physicalZone": "uswest-AZ02"
+                    }
+                  ],
+                  "locationCapabilities": [
```

Comment:
It might make sense to name it just 'capabilities', since it's a subproperty of locationInfo -- or to merge the properties up into the locationInfo object.